components/Editor.js

```diff
@@ -93,6 +93,16 @@ class Editor extends React.Component {
   updateCode = code => this.updateState({ code })
   updateWidth = width => this.setState({ widthAdjustment: false, width })
 
+  calculateExportImageBaseWidth = offsetWidth => {
```

Comment:
@hatsu38 sorry for the delay. Unfortunately, I don't want to do any complex calculation here. 

I would prefer for the editor window to go off screen on smaller displays and have this function just capture what is there than try and make the editor fit on smaller screens and do this calculation after the fact. 

Can we just make the editor go off screen on mobile, but allow the user to easily scroll horizontally?

Reply:
@mfix22 Thank you for your comment.
I will fix it at a later date and push it.

Reply:
@hatsu38 no rush on this at all, but just wanted to check in here to see if you have any updates 🙂

Reply:
Sorry, I have not been able to solve this problem recently due to lack of time

Reply:
No worries at all @hatsu38! I'm going to close this in the meantime, but feel free to re-open it whenever, or open a new PR anytime.